UI/Web/src/app/manga-reader/_components/manga-reader/manga-reader.component.html

```diff
@@ -278,6 +276,13 @@
                      min="10" max="100" step="1" formControlName="darkness">
             </div>
 
+            <div class="col-md-6 col-sm-12">
+              <label for="page-fitting-slider" class="form-label">{{t('width_override')}}:
+                  {{ (widthOverrideLabel$ | async) ? (widthOverrideLabel$ | async) : t('width_override_off') }}
```

Comment:
We can probably re-write this completely as:
@If (widthOverrideLabel$ | async; as widthOverrideLabel) {
{{ widthOverrideLabel ? widthOverrideLabel : t('off') }}
}